pinot-common/src/main/java/com/linkedin/pinot/common/metadata/ZKMetadataProvider.java

```diff
@@ -323,7 +325,7 @@ public static Schema getTableSchema(@Nonnull ZkHelixPropertyStore<ZNRecord> prop
     return resultList;
   }
 
-  public static List<LLCRealtimeSegmentZKMetadata> getLLCRealtimeSegmentZKMetadataListForTable(
+  public static @NotNull List<LLCRealtimeSegmentZKMetadata> getLLCRealtimeSegmentZKMetadataListForTable(
```

Comment:
We usually put @NonNull in front of the method declaration as a separate line (refer to https://www.programcreek.com/java-api-examples/?api=javax.annotation.Nonnull)

Reply:
Done